code/modules/clothing/suits/marine_armor/_marine_armor.dm

```diff
@@ -415,6 +426,17 @@
 	armor_variation = 0
 	specialty = "M3 pattern smooth marine"
 
+/obj/item/clothing/suit/storage/marine/medium/Initialize(mapload)
+	. = ..()
+	if(SSticker.mode && MODE_HAS_FLAG(MODE_FACTION_CLASH))
+		armor_bullet = CLOTHING_ARMOR_HIGH
+		armor_bomb = CLOTHING_ARMOR_MEDIUM
+		armor_internaldamage = CLOTHING_ARMOR_HIGH
+	else
```

Comment:
Generally every else you added is redundant, the stats are already set to be like that